### PR TITLE
data-test-id plugin integration

### PR DESCRIPTION
**Pull request by rwpswami**
@tclindner I am not sure about this integration. Could you please help me in this?
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +55,7 @@
   },
   "peerDependencies": {
     "eslint": "^7.0.0",
+    "eslint-plugin-test-id": "^1.0.2",
```
Comment (tclindner) [inline package.json:58]:
```suggestion
    "eslint-plugin-test-id": "^1.0.0",
```

**Review by tclindner — COMMENTED** [review]
Thank you, @rwpswami!! I messed around with the failure and I'm not sure what is going on. I'll spend some time on it this weekend and see if I can figure out why it isn't registering the rule.

**Comment by tclindner** [comment]
@rwpswami I didn't have any luck. I think it might be something related to relative paths? I tried a few other plugins to see if I'd hit the same issue and I didn't have any issues.

**Comment by rwpswami** [comment]
> @rwpswami I didn't have any luck. I think it might be something related to relative paths? I tried a few other plugins to see if I'd hit the same issue and I didn't have any issues.

I think you are right. This might be related to path. So I have released new version with related paths. Also I have added facility to integrate the plugin as extension. So we don't have to add rules separately.
All test cases have passed now 🎉 

Thanks for your time. I really appreciate it.

File: index.js [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
-const testId = require.resolve('./rules/test-id');
 
 module.exports = {
-  extends: ['plugin:vue/recommended', a11y, vue, vuex, testId],
+  extends: ['plugin:vue/recommended', a11y, vue, vuex, 'plugin:test-id/recommended'],
   parserOptions: {
     parser: 'babel-eslint',
   },
-  plugins: ['vue', 'vuejs-accessibility', 'test-id'],
+  plugins: ['vue', 'vuejs-accessibility'],
```
Comment (tclindner) [inline index.js]: Thanks for adding extends support, @rwpswami 🎉 I think we still want to list the plugin in addition to the extends section. Would you mind adding it back?

```suggestion
  plugins: ['vue', 'vuejs-accessibility', 'test-id'],
```

Comment (rwpswami) [reply index.js]: Yes sure. Added back.

Comment (rwpswami) [reply index.js]: @tclindner Please take a look

**Review by tclindner — APPROVED** [review]
Yay! Nice work, @rwpswami 🙌

**Comment by tclindner** [comment]
I will get a new version out today!

**Pull request merged**